app/core/api/views.py

```diff
@@ -128,6 +131,12 @@ class RecurringEventViewSet(viewsets.ModelViewSet):
     serializer_class = RecurringEventSerializer
 
 
+class PracticeAreaViewSet(viewsets.ModelViewSet):
```

Comment:
This is missing the OpenAPI doc strings for the endpoint.
The @extend_schema_view block